Terminal.Gui/Core/Border.cs

```diff
@@ -145,7 +146,7 @@ void Border_BorderChanged (Border border)
 			/// </summary>
 			/// <param name="border">The border.</param>
 			/// <param name="title">The title.</param>
-			public ToplevelContainer (Border border, string title = null)
+			public ToplevelContainer (Border border, ustring title = null)
```

Comment:
If the user pass as text like "My title" won't fire no errors. If the user pass a string variable, then yes it will raise an exception. The `ustring` type give more options to handle. But this a sealed class that is used by the `Border` class. Though it can be used by others users I don't believed so many users use it. But let @tznind saying something about this, please.

Reply:
Another problem is if I can't change this we can´t show the Title in the border frame.

Reply:
@tznind I think you use this. What you think about the changes, please?

Reply:
I'm not familiar with the `ToplevelContainer` class.  I only really use Border property itself on classes.  

That said, could we add a constructor overload to prevent this being breaking? e.g.

```csharp
public ToplevelContainer (Border border, string title = null)
  : this(border,(ustring)title)
{
}
public ToplevelContainer (Border border, ustring title)
{
...
}
```
I'm on holiday so can't test the above, maybe cast isn't possible but the general idea might be valid?

Reply:
Constructor overload throw `'The call is ambiguous between'` exception, but I tweaked in the Initialize method with this expression `Title = (title != null ? (ustring)title : "")`.
Thanks for the help and a good holiday.